noirup

```diff
@@ -62,10 +64,19 @@ main() {
     RUSTFLAGS="-C target-cpu=native" ensure cargo build --release $FEATURES # need 4 speed
 
     # Remove prior installations if they exist
-    rm -f "$NARGO_BIN_DIR/nargo"
+    for bin in "${BINARIES[@]}"; do
+      rm -f "$NARGO_BIN_DIR/$bin"
+    done
 
-    # Move the binary into the `nargo/bin` directory
-    ensure mv "$PWD/target/release/nargo" "$NARGO_BIN_DIR/nargo"
+    # Move binaries into the `.nargo/bin` directory
+    for bin in "${BINARIES[@]}"; do
+        # Move the binary only if it exists.
+        # This ensures compatibility with older versions of Noir, 
+        # which may not include all binaries listed.
+        if [ -f "$PWD/target/release/$bin" ]; then
+            ensure mv "$PWD/target/release/$bin" "$NARGO_BIN_DIR/$bin"
+        fi
```

Comment:
Can you clear any old binaries in the bin directory? Currently you could install a new version of nargo and get a profiler then switch back to an older version which doesn't have the profiler, and then you've got incompatible noir tooling installed.

Reply:
Ah, I missed you're doing this above.